zstash/globus_utils.py

```diff
@@ -0,0 +1,273 @@
+from __future__ import absolute_import, print_function
+
+import configparser
+import json
+import os
+import os.path
+import re
+import socket
+import sys
+from typing import Dict, List, Optional
+
+from globus_sdk import (
+    NativeAppAuthClient,
```

Comment:
Now we're using the Globus client instead of the `fair_research_login` one, which means 1) it will be easier to ask the Globus support team for help in the future, and 2) we can close #253 as irrelevant.